packages/flutter_tools/templates/app/linux.tmpl/my_application.cc.tmpl

```diff
@@ -3,7 +3,10 @@
 #include <flutter_linux/flutter_linux.h>
 
 #include "flutter/generated_plugin_registrant.h"
-#include "window_configuration.h"
+
+const char *kFlutterWindowTitle = "{{projectName}}";
+const unsigned int kFlutterWindowWidth = 1280;
+const unsigned int kFlutterWindowHeight = 720;
```

Comment:
Let's just inline all of these; once they aren't their own file I don't think we gain anything by separating them from the point where they are used.